Python/robodk/__init__.py

```diff
@@ -24,10 +24,15 @@
 
 #==================================
 # REMOVE BELOW after the adoption period
-from . import robolink, robomath, robodialogs, robofileio
-from .robomath import *
-from .robodialogs import *
-from .robofileio import *
+import sys
+if sys.version_info[0] < 3:
+    # Python 2 does not like the circular dependency
+    pass
```

Comment:
It used to work fine with 2.7. I think someone added a circular dependency afterwards, that's what we should fix.